### DISCUSSION TITLE
New  View

### DISCUSSION THREAD

**Discussion by dopme-io** [community/community#8110] [Projects and Issues] [answered]
Because the new view replicates the previous one. There would be no way to create a new view, without replicating the previous view?

**Comment by azenMatt** [comment, marked as answer]
Hi @DaniloOPinheiro - there are two ways to create a new view: 

1 - As a duplication of an existing view - this is performed via the view menu

![image](https://user-images.githubusercontent.com/7584089/143465293-c5568cd4-6913-4b98-ad41-5aa9c9283e3e.png)

2 - As a 'fresh' view, which just shows the default fields you get when you start a new project - this is performed via the `+ new view` option at the end of the view menu

![image](https://user-images.githubusercontent.com/7584089/143465343-8a737016-9cdf-4e34-b6f6-12bc3cd80f62.png)

**Reply by devinci** [reply]
New view as suggested seems to duplicate the initial view and mirrors changes.

**Reply by harshit-30** [reply]
While adding columns to the existing view, it reflects that it has been added, but they disappear after refreshing.

**Reply by rohit1018** [reply]
not working ... this issue still persists..

**Comment by rohit1018** [comment]
this issue still persists..
